h2o-core/src/main/java/water/api/RequestServer.java

```diff
@@ -665,7 +665,7 @@ private static NanoResponse serveError(H2OError error) {
 
   private static NanoResponse redirectToFlow() {
     NanoResponse res = new NanoResponse(HTTP_REDIRECT, MIME_PLAINTEXT, "");
-    res.addHeader("Location", "/flow/index.html");
+    res.addHeader("Location", H2O.ARGS.context_path + "/flow/index.html");
```

Comment:
@mdymczyk chatted with @mmalohlava and @lo5 this morning. our consensus is that it should redirect from i.e. `localhost:54321` to `localhost:54321/H2O/flow/index.html` when the `context_path` is set to `H2O`